data/archiver/aws_s3_client.py

```diff
@@ -0,0 +1,66 @@
+import os
+import boto3
+import logging
+from data.archiver.dataclass import DataArchiverRequest, DataArchiverResult
+from data.archiver.file_transfer import FileTransfer, TransferProgress, transfer
+from data.archiver.config import AWS_S3_REGION, AWS_S3_BUCKET, AWS_ACCESS_KEY, AWS_SECRET_KEY
+
+
+class AwsS3:
+
+    def __init__(self):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.INFO)
+
+    def get_files(self, req: DataArchiverRequest, res: DataArchiverResult):
+        bucket = self.new_session().resource('s3').Bucket(AWS_S3_BUCKET)
+
+        fs = []
```

Comment:
Is `fs` short for `files`?
Shorthand for variable names is very subjective, thus making it difficult to read and understand.

Reply:
Code refactored